src/pudl/metadata/resources/__init__.py

```diff
@@ -261,10 +259,7 @@
             "standard_so2_percent_scrubbed",
             "data_maturity",
         ],
-        "keep_cols": ["data_maturity"],
     },
-    # utilities must come after plants. plant location needs to be
-    # removed before the utility locations are compiled
```

Comment:
hm is this still the case though?

Reply:
It's not, since I'm now specifically removing the problem columns from the utility table inputs prior to doing the harvesting. Which is a bit jank, but allows all of the harvesting processes to run independent of each other in parallel.